tests/cpp/tts/test_magpietts_file.cpp

```
@@ -243,13 +262,52 @@ main(int argc, char** argv) {
+    std::vector<std::vector<int32_t>> token_chunks;
+    if (!params.text.empty()) {
+        try {
+            tts::MagpieNativeTokenizer tokenizer(params.tokenizer_model_dir);
+            const tts::MagpieTokenizationResult tokenized =
+                tokenizer.tokenize(params.text, params.language_code);
+            params.tokens = tokenized.tokens;
+            token_chunks.reserve(tokenized.chunks.size());
+            for (const auto& chunk : tokenized.chunks) {
+                token_chunks.push_back(chunk.tokens);
+            }
+            std::fprintf(
```

Clang (14.0.6)

[warning] do not call c-style vararg functions

(cppcoreguidelines-pro-type-vararg)